src/libraries/System.Private.CoreLib/src/System/Byte.cs

```diff
@@ -313,11 +313,11 @@ static byte IBinaryInteger<byte>.PopCount(byte value)
             => (byte)BitOperations.PopCount(value);
 
         [RequiresPreviewFeatures]
-        static byte IBinaryInteger<byte>.RotateLeft(byte value, byte rotateAmount)
+        static byte IBinaryInteger<byte>.RotateLeft(byte value, int rotateAmount)
```

Comment:
This was a mistake in the interface definition?

Reply:
Yesn't.

Ideally both `rotate` and `shift` operators would take `TSelf` on the RHS. However, `shift` operators require https://github.com/dotnet/csharplang/issues/4666 to go in and so `Rotate` should match until it does.